INDEX.md

```diff
@@ -6,3 +6,4 @@
 | [`identity-accounts-crypto`](./packages/identity-accounts-crypto) | Crypto accounts linked to your DID  | ![npm version](https://img.shields.io/npm/v/@datamodels/identity-accounts-crypto.svg) |
 | [`identity-accounts-web`](./packages/identity-accounts-web)       | Accounts linked to your DID         | ![npm version](https://img.shields.io/npm/v/@datamodels/identity-accounts-web.svg)    |
 | [`identity-profile-basic`](./packages/identity-profile-basic)     | Basic profile information for a DID | ![npm version](https://img.shields.io/npm/v/@datamodels/identity-profile-basic.svg)   |
+| [`social-connection`](./packages/social-connection)     | Social connection information for a DID | ![npm version](https://img.shields.io/npm/v/@datamodels/social-connection.svg)   |
```

Comment:
Should this be `social-connections` perhaps? Seems like the schema represent multiple connections, not just one connection.

Reply:
thanks for suggestions ! changed

Reply:
changed :)